.github/workflows/ci.yml

```diff
@@ -337,6 +337,8 @@ jobs:
           - retype-parents
           - provider-maven
           - provider-gradle
+          - package-add
+          - convert
```

Comment:
This test is skipped with `t.Skip`, but let's include it here so we don't forget about it.

I had a quick look at unskipping the convert test, but it's failing because it needs an azure subscription ID. Maybe we can change this to use `random` instead, to make this test easily runnable locally and in CI.

We probably want to revise this whole testing setup one day.

Reply:
Yeah we should. I didn't want to perturb too many things right now. I want to do a change touching package add, so just making sure things are tested right now.

Maybe something for the ops rotation.